main.go

```diff
@@ -123,14 +123,20 @@ func decryptSymmetric(keyName string, ciphertext []byte) ([]byte, error) {
 	// Call the API.
 	resp, err := client.Decrypt(ctx, req)
 	if err != nil {
+		log.Fatal("Unable to decrypt.")
```

Comment:
as an FYI - `log.Fatal` calls `os.Exit`, so we won't end up returning `nil, err` here. I think that we could get away with this being a `log.Print`, since it looks like the callers of `decryptSymmetric` check the returned error and pass it to `log.Fatal` anyways. I guess it just depends on whether you want the decrypt error context logged, or just this message.